backend/src/main/java/ca/bc/gov/app/dto/CodeDescrDTO.java

```diff
@@ -0,0 +1,13 @@
+package ca.bc.gov.app.dto;
+
+import java.util.Date;
+
+
+public record CodeDescrDTO(
```

Comment:
@paulushcgcj let's please use the suffix Dto instead of DTO. I know it is an acronym, but let's please use Dto.